### DISCUSSION TITLE
Create GitHub organization for Oshun

### DISCUSSION THREAD

**Discussion by rwp0** [Perl-Oshun/oshun#23] [Ideas]
After (public) or before (as private) opening the repository to the general public.

- [ ] Create a profile photo for the organization to be shown on members' profiles as a badge
- [ ] Distribute work to different repositories in the future (code, spec, gist, etc)

What are the thoughts?

@Ovid we (as users) can get more involved in the organization/management of Oshun

**Comment by Ovid** [comment]
I'd rather keep the specification to a single repository (this one), but I suspect any implementation in Perl itself will not be here. Not even sure if anyone will figure out how to cleanly implement Oshun outside of core Perl. 

That being said, happy to turn this into an organization later and get more people involved and let them have a badge.

**Comment by rwp0** [comment]
> That being said, happy to turn this into an organization later and get more people involved and let them have a badge.

Great! The other option is to collect your "Perl Enhancement" work (`Cor` included) in a more general umbrella organization.

**Comment by Ovid** [comment]
This is now part of https://github.com/Perl-Oshun/. I confess I don't have much experience with github organizations, so perhaps it's not entirely what you were looking for, but I'm trying :)

**Reply by Ovid** [reply]
And I think "Oshun" can be a more general work for the other stuff to enhance Perl.
